rules/opentitan/cc.bzl

```diff
@@ -192,8 +192,8 @@ def _build_binary(ctx, exec_env, name, deps, kind):
     manifest = get_fallback(ctx, "file.manifest", exec_env)
     rsa_key = get_fallback(ctx, "attr.rsa_key", exec_env)
     spx_key = get_fallback(ctx, "attr.spx_key", exec_env)
-    if manifest and kind != "ram":
-        if not rsa_key:
+    if (manifest or rsa_key) and kind != "ram":
+        if not (manifest and rsa_key):
```

Comment:
We should also emit an error if there is an spx_key but not a manifest or rsa key.

Ok to do in a follow-up PR.